service/src/java/org/apache/hive/service/server/HiveServer2.java

```diff
@@ -173,6 +181,7 @@ public class HiveServer2 extends CompositeService {
   private ZooKeeperHiveHelper zooKeeperHelper = null;
   private ScheduledQueryExecutionService scheduledQueryService;
   private ServiceContext serviceContext;
+  public static final String HS2_WEBUI_ROOT_URI = "/hiveserver2.jsp";
```

Comment:
this should go right after the other public static fields above

Reply:
Done. Thanks.

Reply:
thanks!